alfresco-dynamic-extensions-repo/build.gradle

```diff
@@ -1,6 +1,13 @@
 subprojects {
     group "eu.xenit"
 
+    sonarqube {
```

Comment:
> To analyze a project hierarchy, apply the SonarQube plugin to the root project of the hierarchy. Typically (but not necessarily) this will be the root project of the Gradle build. Information pertaining to the analysis as a whole has to be configured in the sonarqube block of this project. Any properties set on the command line also apply to this project.
> -- https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner+for+Gradle

So, this configuration block only has to be applied to the root project, not to all subprojects

Reply:
@vierbergenlars Ready for review